packages/manager/core/generator/app/conditional-templates/dashboard/index.tsx.hbs

```diff
@@ -0,0 +1,53 @@
+import React, { Suspense, } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Await,
+  defer,
+  useLoaderData,
+} from 'react-router-dom';
+
+import Breadcrumb, {
+  BreadcrumbHandleParams,
+} from '@/components/Breadcrumb';
+
+export function breadcrumb({ params }: BreadcrumbHandleParams) {
+  return params.serviceName;
+}
+
+import { {{dashboardEndpoint}} } from '@/api';
+
+export function loader(params: any) {
+  const { serviceName } = params.params
+  return defer({
+    services: {{dashboardEndpoint}}({serviceName}),
+  });
+}
+
+export default function Dashboard() {
+  const { t } = useTranslation('{{appName}}/dashboard');
+   const data = useLoaderData() as Record<string, unknown>;
+
+  return (
+    <div>
+      <Breadcrumb/>
+      <h1>{t('title')}</h1>
+      <div>Dashboard page</div>
```

Comment:
**nitpick:** Can be loaded from the translation files (packages/manager/core/generator/app/conditional-translations/dashboard/Messages_fr_FR.json)

Reply:
👍